src/site/topics/exceptions/exceptions.rst

```diff
@@ -110,7 +110,6 @@ Catching Exceptions
 * If someone is using the ``ArraySortedBag`` implementation two years from now, it's not possible to know what they should do to manage the exceptional situations
 * The users of the ``ArraySortedBag`` class will need to deal with them as they see fit
 
-    * What should be done if calling ``remove`` on an empty ``ArraySortedBag``?
```

Comment:
Things would need to be rewritten a little --- It needs to be 
"What should be done if calling ``removeFirst`` on an empty ``ArraySortedBag``